e2e/paths/loginAsHmctsAdmin_test.js

```diff
@@ -12,12 +12,45 @@ Before(async (I, caseViewPage, submitApplicationPage) => {
   I.signOut();
 });
 
-Scenario('HMCTS admin can login and add a FamilyMan case number to a submitted case', (I, caseViewPage, loginPage, caseListPage, enterFamilyManPage) => {
+Scenario('HMCTS admin can login and add a FamilyMan case number to a submitted case', (I, caseViewPage, loginPage, enterFamilyManPage, enterOtherProceedingsPage) => {
```

Comment:
I am bit uncertain whether having everything in one test is best approach but in my might it will be subject to my refactoring.

I am also bit worried that tests focus more on checking that event summary box is in place instead of interacting with edit pages itself. Test for example does not test that amending child details is possible, it just checks whether action is available and whether such action can be completed with providing event name and summary.